tests/modules/collaborations/resources/test_collaboration_notification.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import tests.modules.collaborations.resources.utils as collab_utils
+import tests.modules.notifications.resources.utils as notif_utils
+from tests import utils as test_utils
+
+
+# Full sequence, create the collaboration, validate that the notification is received, approve the view aspect
+# Request edit, validate that notification is received, approve edit.
+def test_edit_collaboration(flask_app_client, researcher_1, researcher_2, db, request):
```

Comment:
@brmscheiner I created a test that did the full sequence as you asked

Reply:
thanks for doing this!